code/API_definitions/qod-api.yaml

```
@@ -222,7 +222,67 @@ paths:
         - oAuth2ClientCredentials: []
         - threeLegged:
             - "qod-sessions-write"
-
+    get:
+      tags:
+        - QoS sessions
+      summary: Get all active sessions information
+      description: Get information about all active sessions that were created by the client
```

Here, we should indicate that the returned sessions will be those allowed to be discovered with the provided access token, which will depend on the type of access token provided, 2-legged or 3-legged. This has to be clarified in the implementation guidelines and aligned with Identity&Consent Management guidelines, but to progress with the spec we could add something like:

```suggestion
      description: Get information about all active sessions authorized to be retrieved by the provided access token.
```